### ISSUE TITLE
[A11y] [Issue 13] ARIA attributes have invalid values within chat

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#3139]
# 13 Severity 2/3 – ARIA attributes have invalid values within chat

> This is filed to https://github.com/microsoft/BotFramework-WebChat/issues/3139.

## Background

Accessibility Insights issues

1. Element path: #cdc-covid19-healthbot;.css-sph49o
Snippet: <div aria-labelledby="webchat__toaster__header__1adkb" aria-live="polite" aria-relevant="additions text" class="css-sph49o css-6wwnjx webchat__toaster" role="log">
Fix all of the following: Invalid ARIA attribute value: aria-labelledby="webchat__toaster__header__1adkb"
1. Element path: #cdc-covid19-healthbot;#webchat__toaster__list__3ugck
Snippet: <ul aria-labelledby="webchat__toaster__header__1adkb" class="webchat__toaster__list" id="webchat__toaster__list__3ugck" role="region"></ul>
Fix all of the following: Invalid ARIA attribute value: aria-labelledby="webchat__toaster__header__1adkb"

## Investigation notes

The element pointed by `aria-labelledby` is not available. This should be fixed in a recent PR but might not have pulled into 4.8.1.

[Bug]

**Comment by compulim** [comment]
This is fixed in #3098.
